functions/prompts.py

```
@@ -0,0 +1,96 @@
+"""
+Centralized store for all LLM prompts used in the application.
+"""
+
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)